scalableminds/webknossos
# Reject dataset uploads if organization storage quota is exceeded

### Steps to test:
- set `reportUsedStorage.enabled = true` for your datastore in application.conf (note that if your organization is super new, you may want to lower rescanInterval as well so that you don’t have to wait 24 hours. initial data sets lastScanTime to creationTime.)
- upload a dataset, should still work
- in the local db, limit your organization’s includedStorage field, retry, should be rejected

### Issues:
- fixes #6808

------
